entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -9,99 +9,218 @@ uid: core/get-started/aspnetcore/new-db
+
+# [Visual Studio](#tab/visual-studio)
+
+Choose this tab for Visual Studio 2017 instructions.
 
 [View this article's sample on GitHub](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb).
 
+# [.NET Core CLI](#tab/netcore-cli)
+
+Choose this tab to use the .NET Core CLI with any code editor on Windows, macOS, or Linux.
+
+[View this article's sample on GitHub](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCoreNewDb.Sqlite/EFGetStarted.AspNetCore.NewDb).
+
+---
+ 
 ## Prerequisites
 
 Install the following software:
 
-* [Visual Studio 2017 15.7](https://www.visualstudio.com/downloads/) with these workloads:
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.7 or later](https://www.visualstudio.com/downloads/) with these workloads:
   * **ASP.NET and web development** (under **Web & Cloud**)
   * **.NET Core cross-platform development** (under **Other Toolsets**)
 * [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
 
-## Create a new project in Visual Studio 2017
+# [.NET Core CLI](#tab/netcore-cli)
+
+* [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
+
+---
+
+## Create a new project
+
+# [Visual Studio](#tab/visual-studio)
 
 * Open Visual Studio 2017
 * **File > New > Project**
-* From the left menu select **Installed > Visual C# > .NET Core**.
+* From the left menu, select **Installed > Visual C# > .NET Core**.
 * Select **ASP.NET Core Web Application**.
 * Enter **EFGetStarted.AspNetCore.NewDb** for the name and click **OK**.
 * In the **New ASP.NET Core Web Application** dialog:
-  * Ensure the options **.NET Core** and **ASP.NET Core 2.1** are selected in the drop down lists
+  * Make sure that **.NET Core** and **ASP.NET Core 2.1** are selected in the drop-down lists
   * Select the **Web Application (Model-View-Controller)** project template
-  * Ensure that **Authentication** is set to **No Authentication**
+  * Make sure that **Authentication** is set to **No Authentication**
   * Click **OK**
 
-Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to your project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to the project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* At a command prompt run the following command:
```

Comment:
Run the following command to create a new MVC app.

They know to run it at the command prompt - besides, it's not called command prompt on macOS and linux.